java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/SymbolicValue.java

```diff
@@ -104,6 +105,14 @@ public void computedFrom(List<SymbolicValue> symbolicValues) {
     // no op in general case
   }
 
+  public void computedFromSymbols(List<ProgramState.SymbolicValueSymbol> computedFrom) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1186)